sunpy/conftest.py

```diff
@@ -13,12 +14,25 @@
 else:
     matplotlib.use('Agg')
 
+# isort:imports-firstparty
 import sunpy.tests.helpers
 from sunpy.tests.hash import HASH_LIBRARY_NAME
 from sunpy.tests.helpers import new_hash_library, generate_figure_webpage
 from sunpy.util.exceptions import SunpyDeprecationWarning
 
 
+if importlib.util.find_spec('asdf') is not None:
+    from asdf import __version__ as asdf_version
+    # Skip schema tests on astropy LT 3.1 because the coordinate schema only
+    # work for 3.1 The tags are already not registered on LT 3.1
+    from astropy import __version__ as astropy_version
+    if (parse_version(asdf_version) >= parse_version('2.0.0')
```

Comment:
I'm not sure what your requirements are for `asdf`, but Astropy 3.1 requirest `asdf` 2.3.0.